src/@types/index.ts

```diff
@@ -0,0 +1,8 @@
+export type NameTheFrameSettings = {
+  description: string;
+};
+
+export enum NameTheFrameSettingsNames {
```

Comment:
Convention we use for enum name is singular.
```suggestion
export enum SettingName {
```